docs/decisions/2023-06-05_github_merge_queue.md

```diff
@@ -0,0 +1,49 @@
+# 2023-06-05 GitHub Merge Queue
+
+* status: proposed
+* date: 2023-06-05
+* authors: Brian Samek
+
+## Context and Problem Statement
+
+* [Epic](https://go/github-merge-queue)
+* [Scope](https://go/github-merge-queue-scope)
+* [Design](https://go/github-merge-queue-design)
+
+GitHub merge queue integrates natively with GitHub Actions. Integration with
```

Comment:
Yeah. I meant to imply "since the documentation is pretty sparse for external CI systems, I'm writing this ADR to supplement it". Like I've already discovered that listening for branches of a particular name, like the docs suggest, might not be correct.

Reply:
I'd be curious to learn more about "listening for branches of a particular name", but no rush (and doesn't have to be here)